Source/Editor/DualityEditor/DualityEditorApp.cs

```diff
@@ -849,116 +854,118 @@ public static void ReadPluginSourceCodeContentData(out string rootNamespace, out
 		}
 		public static void InitPluginSourceCode()
 		{
-			// Create solution file if not existing yet
-			if (!File.Exists(EditorHelper.SourceCodeSolutionFile))
+			// Check if the solution file has to be created		
+			if (EditorHelper.SourceCodeSolutionFilePath == null)
 			{
+				// Create solution file
 				using (MemoryStream gamePluginStream = new MemoryStream(Properties.GeneralRes.GamePluginTemplate))
 				using (ZipArchive gamePluginZip = new ZipArchive(gamePluginStream))
 				{
 					gamePluginZip.ExtractAll(EditorHelper.SourceCodeDirectory, false);
 				}
-			}
```

Comment:
Creating the solution file and initializing an existing solution file to match a project have been two independent operations, and they should remain independent for the time being, so the followup code should continue to be outside of the initial `if` block. It's possible that the current `Create Project` dialog relies on this.

Reply:
Yup my code broke the ```Create Project``` dialog. Will fix it.